_posts/2021-12-23-Fortran-lang-2021-in-review.md

```diff
@@ -0,0 +1,350 @@
+  the command line.
+* [fpm-man](https://github.com/urbanjost/fpm-man):
+  Adds the `fpm man` command for displaying man-style help pages about Fortran
+  intrinsics and other language features.
+
+At the time of writing, there are almost 200 projects now using fpm.
+If you haven't tried fpm yet, please do!
+It has truly been a game-changing tool for many of us.
+
+## LFortran
+
+<center>
+<a href="https://lfortran.org">
+<img src="{{ site.baseurl }}/assets/img/Fortran-lang-2021-in-review/lfortran-logo-300x300.png">
+</a>
+</center>
+
+Though not technically a Fortran-lang project,
+[LFortran](https://lfortran.org) has been growing close to the Fortran-lang
+community and projects largely thanks to its creator and lead developer 
+[Ondřej Čertík](https://github.com/certik) also being one of the founding
+members of Fortran-lang.
+LFortran has been developing rapidly this year and was released as a
+[Minimum Viable Product (MVP)](https://lfortran.org/blog/2021/09/lfortran-minimum-viable-product-mvp/) in September.
+LFortran currently parses all of Fortran 2018 and compiles a significant
+subset of the language.
+If you haven't tried LFortran yet, please do and let us know how it can best
+improve for your use case.
+You can help speed up LFortran's development by contributing code and/or
+documentation directly, or by donating funds to the project via
+[NumFOCUS](https://numfocus.org/project/lfortran).
+
+Follow LFortran on Twitter at [@lfortranorg](https://twitter.com/lfortranorg).
+
+## fortran-lang.org, Discourse, and social media
+
+A major addition to the Fortran website this year is the
+[Fortran Best Practices mini-book](https://fortran-lang.org/learn/best_practices).
+This is a port and an update to the well-known
+[fortran90.org](https://fortran90.org) by
+[Ondřej Čertík](https://github.com/certik).
+It provides a plethora of tips on how to write simple and idiomatic Fortran
+for numerical tasks, and how to avoid some common gotchas.
+Give it a read and let us know how it can be improved.
+
+The [Fortran Discourse](https://fortran-lang.discourse.group) is as active
+as ever, with new users joining daily.
+There are currently 338 active users out of a total of 535 registered users.
+The Fortran Discourse is a great place to ask for help with Fortran code, post
+a Fortran job opening, and discuss anything Fortran-related.
+
+We also continue to release the monthly newsletter where we
+document the progress month-by-month, as well as post about any notable events
+in the Fortran world.
+This newsletter is a great way to stay up to date with Fortran-lang.
+If you're also on Twitter, follow our account
+[@fortranlang](https://twitter.com/fortranlang) for daily bite-size news and
+updates, as well as the new [@FortranTip](https://twitter.com/FortranTip)
+account, managed by [Beliavsky](https://github.com/beliavsky), which brings
+daily Fortran tips to your Twitter feed.
+
```

Comment:
Proposition for the Fortran Monthly calls:

```suggestion

Finally, a call is orgnanized monthly to discuss issues and topics related to all
Fortran-lang projects. The Fortran Monthly calls are mainly advertised on Fortran Discourse,
and are open to everyone.
Don't hesitate to join the call, to meet other Fortran-lang participants and to take part to the different discussions.
```

Reply:
Thanks, that's good. What do you think about this (slightly re-written)?

```suggestion

Finally, we meet on monthly video calls to discuss issues and topics related to all
Fortran-lang projects.
The calls are advertised on Fortran Discourse and are open to everyone.
Don't hesitate to join the call to meet other Fortran-lang participants and take part
in different discussions.
```

Reply:
> What do you think about this (slightly re-written)?

LGTM. Thank you!